libbpf-rs/tests/test.rs

```diff
@@ -647,3 +649,40 @@ fn test_object_task_iter() {
     // Check for init
     assert!(items.iter().any(|&item| item.pid == 1));
 }
+
+#[test]
+fn test_create_map_and_pin() {
+    bump_rlimit_mlock();
+
+    let mut map = Map::create_map(
```

Comment:
I'm seeing this test fail in CI. 
```

running 1 test
test test_create_map_and_pin ... FAILED

failures:

---- test_create_map_and_pin stdout ----
thread 'test_create_map_and_pin' panicked at 'assertion failed: `(left == right)`
  left: `-1`,
 right: `0`: Setting RLIMIT_MEMLOCK failed with errno: 1', libbpf-rs/tests/test.rs:48:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    test_create_map_and_pin

```

Would you mind taking a look?

Reply:
OK, I will fix the CI error.

Reply:
It seems that the original test function `test_create_map_and_pin()` needs sudo operation and should skip in `Run tests`(`./github/rust/yml`).
I rename the test function to `test_object_map_create_and_pin()` which start with `test_object_` prefix to adapt the CI configurations.